popolo/importers/popit.py

```diff
@@ -407,8 +407,10 @@ def generate_membership_id(membership_data):
         result.end_date = membership_data.get('end_date', '')
         if 'legislative_period_id' in membership_data:
             period_data = self.events[membership_data['legislative_period_id']]
-            result.start_date = period_data.get('start_date', '')
-            result.end_date = period_data.get('end_date', '')
+            if not result.start_date:
+                result.start_date = period_data.get('start_date', '')
+            if not result.end_date:
+                result.end_date = period_data.get('end_date', '')
```

Comment:
@wfdd Yeah, it's not quite right, but I think this is a useful default in the absence of support for Popolo events in django-popolo. I'm about to push a rewritten version of this commit which adds a comment explaining that, and I've created this ticket: https://github.com/openpolis/django-popolo/issues/30